src/main/java/com/purbon/kafka/topology/TopologyBuilderAdminClient.java

```diff
@@ -46,17 +47,27 @@ public TopologyBuilderAdminClient(AdminClient adminClient) {
     this.adminClient = adminClient;
   }
 
-  public Set<String> listTopics() throws IOException {
+  public Set<String> listTopics(ListTopicsOptions options) throws IOException {
     Set<String> listOfTopics;
     try {
-      listOfTopics = adminClient.listTopics().names().get();
+      listOfTopics = adminClient.listTopics(options).names().get();
```

Comment:
Could be simplified as
```
    try {
      return adminClient.listTopics(options).names().get();
    } catch (InterruptedException | ExecutionException e) {
      LOGGER.error(e);
      throw new IOException(e);
    }
```

Reply:
true, I just happen to like returns at the end, but is good no extra object will be created to only do the return ;-) good catch, thanks!